libs/cardano-ledger-test/src/Test/Cardano/Ledger/Constrained/Vars.hs

```diff
@@ -680,7 +680,7 @@ markPoolDistrL ::
 markPoolDistrL = nesEsL . esSnapshotsL . ssStakeMarkPoolDistrL . pooldistrHelpL
 
 pooldistrHelpL :: Lens' (PoolDistr c) (Map (KeyHash 'StakePool c) (IndividualPoolStake c))
-pooldistrHelpL = lens unPoolDistr (\_ u -> PoolDistr u)
+pooldistrHelpL = lens unPoolDistr (\x y -> x {unPoolDistr = y})
```

Comment:
Remove this lens in favor of the newly added `poolDistrDistrL`. No need to keep duplicate code around
```suggestion
```